apps/system/js/nfc_handover_manager.js

```diff
@@ -132,6 +132,15 @@
       this.bluetoothStatusSaved = false;
       this.bluetoothAutoEnabled = false;
 
+      // Besides listening to bluetooth-enabled event,
+      // request the adapter at module start to 
+      // avoid the module loading dependency
+      Service.request('Bluetooth:adapter').then((adapter) => {
+        this._adapter = adapter;
+      }).catch((err) => {
```

Comment:
This promise NEVER rejects according to bluetooth.js but I agree we could still put this in just to safeguard.